src/errors.rs

```diff
@@ -1,4 +1,7 @@
-/// Hashing errors.
+/// # Fantom/libHash/errors
```

Comment:
Please provide a descriptive title or feel free to skip the title altogether like in https://rust-lang-nursery.github.io/futures-api-docs/0.3.0-alpha.17/src/futures/lib.rs.html.